gnuradio-runtime/lib/CMakeLists.txt

```diff
@@ -135,14 +140,20 @@ target_link_libraries(gnuradio-runtime PUBLIC
   # INTERFACE/PRIVATE split so users of the library can choose how to link to Python
   # (importantly, extension modules can avoid linking against Python and resolve
   #  their own Python symbols at runtime through the Python interpreter's linking)
-  INTERFACE
-    Python::Module
   PRIVATE
-    Python::Python
-    libunwind::libunwind
-    pybind11::pybind11
+    libunwind::libunwind 
   )
 
+if (ENABLE_PYTHON)
+target_link_libraries(gnuradio-runtime
+  INTERFACE
+  Python::Module
+  PRIVATE
+  Python::Python
+  pybind11::pybind11
```

Comment:
Agree in principle, but isn't this part of the runtime completely independent from Pybind? Why do we need to link it against pybind?

Reply:
The only part that is not independent is the gateway blocks that use the python embedding via pybind.  But linking is a good question since pybind11 is header only.  It complains though if it is not there.  Perhaps a good question for @ryanvolz

Reply:
When you add a "link" to the `pybind11::pybind11` target, what it actually does is add the include paths for pybind11 and python to your target. No actual linking is added.